chain/network/src/peer_manager/peer_store.rs

```diff
@@ -339,6 +339,7 @@ impl PeerStore {
     }
 
     /// Adds a peer into the store with given trust level.
+    #[inline(always)]
```

Comment:
I'd rather not do this, as this adds to visual noise more than the benfit it provides. LLVM is pretty good at inlining, and we are compiling with full lto as well, and in debut it'd actually be beneficial to not triplicate the function.

Or do we know for sure that this reduces codesize or improves performance?